AWSTranscribeStreaming/Internal/AWSSRWebSocketDelegateAdaptor.m

```diff
@@ -115,10 +119,19 @@ - (void)webSocket:(AWSSRWebSocket *)webSocket didFailWithError:(NSError *)error
     });
 }
 
-- (void)webSocket:(AWSSRWebSocket *)webSocket
- didCloseWithCode:(NSInteger)code
-           reason:(NSString *)reason
-         wasClean:(BOOL)wasClean {
+/**
+The websocket disconnected
+
+@param webSocket the web socket that failed
+@param code the error code it closed with
+@param reason why did the socket close
+@param wasClean  did the scoket close cleanly
+*/
+
+- (void)webSocketDisconnected:(AWSSRWebSocket *)webSocket
```

Comment:
As above, why do we need the extra layer of indirection?

Reply:
explained above